test/LocalDataSource.spec.tsx

```diff
@@ -388,6 +389,18 @@ describe('LocalDataSource', () => {
         done();
       });
     });
+
+    it('should return a payload without filters', (done) => {
```

Comment:
Please change the description... doesn't look like GTE

Reply:
Done, @kadosh